### PR TITLE
Add test for admin login feature

### PR DESCRIPTION
**Pull request by rijju-pdl**
example of data table and example table

### CODE REVIEW

File: tests/features/login.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+
+
+
+
+    
+
+
```
Comment (anon-pradip) [inline tests/features/login.feature]:
```suggestion
```

File: tests/stepDefinitions/loginContext.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//datatable
+When('the user logs in with following details:', function (dataTable) {
+  // Write code here that turns the phrase above into concrete actions
+  for(const {email,password} of dataTable.hashes())
+  {
+    console.log(email,password)
+  }
+
+
+
```
Comment (anon-pradip) [inline tests/stepDefinitions/loginContext.js]:
```suggestion
```

File: tests/features/register.feature [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Given I am on registeration page
+# When I register with the following details
+#  | email           | password    |
+#  | rijju@gmail.com | abc@123     |
+#  | ram@gmail.com   | ram@123     |
+
+# Then i should see the confirmation page.
+
+
+
```
Comment (anon-pradip) [inline tests/features/register.feature]: this file should be removed

File: tests/features/login.feature [hunk 4]
```diff
@@ -0,0 +1,44 @@
+Feature: Login
+    As an admin
+    I want to login to the webiste's admin panel
+    So that I can access to the home page
```
Comment (prashant-gurung899) [inline tests/features/login.feature]:
```suggestion
    So that I can manage my products
```

File: tests/features/login.feature [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    Scenario: login with valid credentials
+        When  user "admin" logs in with email "admin@admin.com" and password "admin123456"
+        Then user "admin" should be navigated to the admin panel dashboard.
+
+    Scenario:login with invalid credentials
+        When user "admin" logs in with email "admin@admin.com" and password "hi12345"
+        Then the "Invalid Crediantials" message should be pop in the login page.
+
+
+
```
Comment (prashant-gurung899) [inline tests/features/login.feature]: Always keep two lines in-between scenarios
```suggestion
```

File: tests/features/login.feature [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        | email           | password    |
+        | rijju@gmail.com | abc@123     |
+        | rijju@gmail.com | abc@123     |
+        Then the user should be logged in successfully
+
+    
+     Scenario Outline: login with valid credentials using example table
+        When the user logs in with email "<email>" and password "<password>" .
+        Then the user should be logged in successfully.
+
```
Comment (prashant-gurung899) [inline tests/features/login.feature:14]:
```suggestion
```

File: tests/features/login.feature [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        When  user "admin" logs in with email "admin@admin.com" and password "admin123456"
+        Then user "admin" should be navigated to the admin panel dashboard.
+
+    Scenario:login with invalid credentials
+        When user "admin" logs in with email "admin@admin.com" and password "hi12345"
+        Then the "Invalid Crediantials" message should be pop in the login page.
+
+
+
+    Scenario: login with valid credentials using data table
```
Comment (prashant-gurung899) [inline tests/features/login.feature]:
```suggestion
    Scenario: login with valid credentials
```

File: tests/features/login.feature [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+    Scenario: login with valid credentials using data table 
+        When the user logs in with following details:
+        | email           | password    |
+        | rijju@gmail.com | abc@123     |
+        | rijju@gmail.com | abc@123     |
+        Then the user should be logged in successfully
+
+    
+     Scenario Outline: login with valid credentials using example table
```
Comment (prashant-gurung899) [inline tests/features/login.feature]:
```suggestion
     Scenario Outline: login with valid credentials
```

File: tests/features/login.feature [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+        Then the "Invalid Crediantials" message should be pop in the login page.
+
+
+
+    Scenario: login with valid credentials using data table 
+        When the user logs in with following details:
+        | email           | password    |
+        | rijju@gmail.com | abc@123     |
+        | rijju@gmail.com | abc@123     |
+        Then the user should be logged in successfully
```
Comment (prashant-gurung899) [inline tests/features/login.feature]:
```suggestion
        Then the user should be navigated to the home page
```

File: tests/features/login.feature [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+    Background:
+        Given the user "admin" has navigated to the login page
+
+    Scenario: login with valid credentials
+        When  user "admin" logs in with email "admin@admin.com" and password "admin123456"
+        Then user "admin" should be navigated to the admin panel dashboard.
+
+    Scenario:login with invalid credentials
+        When user "admin" logs in with email "admin@admin.com" and password "hi12345"
```
Comment (prashant-gurung899) [inline tests/features/login.feature]: We use `tries to` phrase in events that wont happen
```suggestion
        When user "admin" tries to log in with email "admin@admin.com" and password "hi12345"
```

**Comment by prashant-gurung899** [comment]
LGTM :+1:

**Pull request merged**